suite-native/module-connect-popup/src/hooks/useConnectPopupNavigation.ts

```diff
@@ -34,33 +34,12 @@ export const useConnectPopupNavigation = () => {
     const featureFlagEnabled = useFeatureFlag(FeatureFlag.IsConnectPopupEnabled);
     const navigation = useNavigation<NavigationProp>();
 
-    const navigateToConnectPopup = useCallback(
-        (url: string) => {
-            if (!featureFlagEnabled) return;
-            if (!isConnectPopupUrl(url)) return;
-            const parsedUrl = Linking.parse(url);
-            navigation.navigate(RootStackRoutes.ConnectPopup, { parsedUrl });
-        },
-        [navigation, featureFlagEnabled],
-    );
+    const url = Linking.useURL();
```

Comment:
Did you tried all cases? I think I played with this hook intially but found some edge case where it doesn't worked correctly but I can't recall it now.

Reply:
After updating the library I didn't see any issue. I looked at what the hook does and it's basically the same thing.